src/main/java/org/folio/rest/impl/NotificationsResourceImpl.java

```diff
@@ -281,6 +282,13 @@ public void postNotify(String lang,
     if (id == null || id.trim().isEmpty()) {
       id = UUID.randomUUID().toString();
     }
+    if (!UuidUtil.isUuid(id)) {
+      Errors valErr = ValidationHelper.createValidationErrorMessage(
+        "id", id, "invalid input syntax for type uuid");
+      asyncResultHandler.handle(succeededFuture(PostNotifyResponse
+        .respond422WithApplicationJson(valErr)));
+      return;
+    }
```

Comment:
Why is this a part of RMB update?

Reply:
Id validation was removed from PostgresClient. In current implementation most of tests were failed, it doesn't return validation error. Current version of PostgresClient uses UUID.fromString() method under the hood. It doesn't check UUID correctly.